ruby/exercises/palindromo.rb

```diff
@@ -7,3 +7,11 @@
   # 'Madam'
 # Ejemplo invalido:
 # 'Hello'
+def pali?(xx)
+##  xx == xx.reverse ? true : false
+xx == xx.reverse
```

Comment:
Hay que cuidar la indentacion cuando escribimos codigo para que las personas que lean el codigo puedan hacerlo mas rapido y mas facil por favor
```suggestion
  xx == xx.reverse
```